app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= bootstrap_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <header class="d-flex justify-content-between align-items-center">
+    <h2 class="p-2" onclick="history.back()"> ⬱ </h2>
+    <h2 class="p-2">REGISTER</h2>
+    <div class="actions p-2">
+      <%= f.submit "Next", class: "btn btn-primary" %>
```

Comment:
- [ ] It would be helpful if you label this form as "LOGIN" since it is used for logging in.

Reply:
The change was did, thanks